### PR TITLE
[docs] post-PR merge fix

### PR DESCRIPTION
**Pull request by stas00**
Reverts incorrect changes merged in https://github.com/huggingface/transformers/pull/15346

### CODE REVIEW

File: docs/source/main_classes/deepspeed.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ won't be possible on a single GPU.
 
 🤗 Transformers integrates [DeepSpeed](https://github.com/microsoft/DeepSpeed) via 2 options:
 
-1. Integration of the core DeepSpeed features via [`Trainer`]. This is everything done for your type
+1. Integration of the core DeepSpeed features via [`Trainer`]. This is everything done for you type
```
Comment (sgugger) [inline docs/source/main_classes/deepspeed.mdx:34]:
```suggestion
1. Integration of the core DeepSpeed features via [`Trainer`]. This is an everything-done-for-you type
```
I don't understand that sentence. If "everything done for you" is to be interpreted as an adjective, it should have the hyphen and an article before, but I'm not sure if this is what you meant.

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

File: docs/source/main_classes/deepspeed.mdx (hunk 1 above)
Comment (stas00) [reply docs/source/main_classes/deepspeed.mdx:34]: Yes, I like your suggestion, @sgugger - thank you!

**Review by ngoquanghuy99 — COMMENTED** [review]
@stas00 Thank you much for reverting my PR!

**Comment by stas00** [comment]
And thanks again for the proposed improvements, @ngoquanghuy99!

**Pull request merged**